src/pages/Flags/Flags.tsx

```diff
@@ -22,7 +23,7 @@ const FlagHeader = () => {
 }
 
 export const Flags = ({ route }: FlagsPageProps) => {
-  const featureFlags = Object.keys(useSelector(selectFeatureFlags))
+  const featureFlags = Object.keys(useAppSelector(selectFeatureFlags)) as Array<keyof FeatureFlags>
```

Comment:
As discussed in-person, this is not referentially stable as every-time `<Flags />` will re-render, a new object reference would be created because of `Object.keys()`. Usually this should live in a derived selector for referential stability, but since this is literally just used for the flags page, it's not a blocker, just nice-to-have.